src/program.js

```diff
@@ -341,6 +341,15 @@ Example: $0 --help run.
         demand: false,
         type: 'string',
       },
+      'firefox-port': {
+        alias: 'P',
+        describe: 'Attach to a running Firefox on this developer tools ' +
+                  'port. To do so, open the developer toolbar and enter ' +
```

Comment:
We should also show these instructions on the console in case a connection cannot be made